src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -1007,7 +1033,28 @@ static inline bool notification_match(reservation_t& res,
         return ret;
       }
     }
-    res.topics.emplace_back(topic_filter.s3_id, topic_cfg, res_id);
+    // load the topic,if there is change in topic config while it's stored in
+    // notification.
+    rgw_pubsub_topic result;
+    const RGWPubSub ps(res.store, res.user_tenant);
+    auto ret = ps.get_topic(res.dpp, topic_cfg.name, result, res.yield,
+                            /*fetch_bucket_mapping=*/false);
+    if (ret < 0) {
+      ldpp_dout(res.dpp, 1) << "ERROR: failed to load topic: " << topic_cfg.name
```

Comment:
we should not log an `ERROR` here. 
e.g. users will be alarmed that there is an issue after topic deletion (or before sync happened).
maybe log `INFO` message?